bb_topic: BBox
rgbd_topic: camera/depth_registered/points
output_topic: filtered_points
